package/lighthouse-crd.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: multiclusterservices.lighthouse.submariner.io
+spec:
+  group: lighthouse.submariner.io
+  version: v1
+  names:
+        kind: MultiClusterService
+        plural: multiclusterservices
+        singular: multiclusterservice
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required: ["clusterServiceInfo"]
```

Comment:
Does this mean `clusterServiceInfo` must have at least one element? If so I don't think we should impose this restriction in case we don't actually delete `MultiClusterService` instances when the last `ClusterServiceInfo` element is removed.

Reply:
Removed the required field.